mitodl/course-catalog
# Add blank=True to nullable fields and add nullable fields

#### What are the relevant tickets?
Closes #28 

#### What's this PR do?
This PR modifies Course and CoursePrice tables to allow for values that match the incoming data.
CoursePrice objects can now handle up to six digit prices.
Courses can how have null values in year, semester, language, and level fields.
blank=True has been added to nullable fields so the serializer correctly validates incoming data.

#### How should this be manually tested?
Open the project in the shell and manually run get_edx_data(). You should change the code to run on catalog 10 (and use the correct id and key for that catalog). The catalog should probably be an environment variable, but I will do that in a different PR.
